rtgui/guiutils.h

```diff
@@ -430,6 +436,22 @@ class MyImageMenuItem : public Gtk::MenuItem
     const Gtk::Label* getLabel () const;
 };
 
+class MyProgressBar : public Gtk::ProgressBar
```

Comment:
Things like this should be done after the Gtk2 branches are dropped I think, otherwise I'll let you merge it with the Gtk3 branch ;) There's no hurry.